packages/shared/src/hooks/streaks/useStreakRecover.ts

```diff
@@ -52,10 +53,9 @@ export const useStreakRecover = ({
   const { updateAlerts } = useAlertsContext();
   const { user } = useAuthContext();
   const client = useQueryClient();
-  const { data, isLoading } = useQuery<StreakQueryData>({
-    queryKey: generateQueryKey(RequestKey.UserStreakRecover),
-    queryFn: async () => await gqlClient.request(USER_STREAK_RECOVER_QUERY),
-  });
+  const {
+    query: { streak_restore: streakRestore },
+  } = useRouter();
```

Comment:
This seems a bit pointless right?
The modal never shows if that router element is not available already or if it didn't come from bootPopup?

Reply:
If modal is opened by BootPopup we do not have this query string. 
Added just for checking if the user actively opened the modal, avoiding toast without context!